docs/organizations/integrations/slack-integration.md

```diff
@@ -0,0 +1,25 @@
+# Organization Slack integration for Security issues
+
+This integration sends notifications to a Slack channel of your choice whenever Codacy detects new critical Security issues in any repository within your organization.
+
+## Installing the Slack integration
+
+To install the Slack integration:
+
+1.  Follow the [instructions on the Slack documentation pages](https://api.slack.com/messaging/webhooks) to add an Incoming Webhook to your Slack workspace and set it to post to a channel of your choice, and then copy the webhook URL.
```

Comment:
Ideally, we should have only one action per step. We have three actions in this step (follow the instructions, set it, copy). We could leave the first together in this step, as they are executed on Slack side, and place the copy in a separate step. WDYT?

```suggestion
1.  Follow the [instructions on the Slack documentation pages](https://api.slack.com/messaging/webhooks) to add an Incoming Webhook to your Slack workspace and set it to post to a channel of your choice.
1.  Copy the webhook URL.
```

Reply:
Sounds good, I'm extending this with another step.